### ISSUE TITLE
Non-static method and This Warning

### ISSUE THREAD

**Issue by APGRoboCop** [wakachamo/PsychoStats#6]
I am getting some Static issue warnings when installing this PStats on my website. It appears this guy got the same problem but found the solution.

"Strict Standards: Non-static method PsychoDB::create() should not be called statically in www/stats/install/common.php on line 81

Warning: Cannot modify header information - headers already sent by (output started at /home3/careersu/public_html/a1nomercy.tk/stats/install/common.php:81) in www/stats/includes/class_session.php on line 236"

http://stackoverflow.com/questions/25486491/please-help-how-can-i-solve-non-static-method-and-this-warning

**Comment by Murphyprecht** [comment]
yeah, but it isn't clear what to do if you're noob. Plain english please?

Strict Standards: Non-static method PsychoStats::create() should not be called statically in /includes/common.php on line 102

**Comment by APGRoboCop** [comment]
Actually we've just some fixes to PsychoStats and added some new additional features, thanks to Drek.

**Comment by stamepicmorg** [comment]
> some fixes

hello! maby `fork` + `pull requests`?

**Comment by Drek282** [comment]
At this time I don't do github.  If anyone wants to take what I've done and merge it with this project, or any other, I have no problem with that.  I'm just not up to being actively involved in, or responsible for, a github project.  But I am actively working on Psychostats right now.  I'm trying to add support for Firearms and TFC events.  I've made some headway but I'm banging my head against a wall right now.  This is mostly about learning something new.  I know something about bash scripting, and not that much about php and perl, but I'm learning.  Stormtrooper is a tough teacher...

What I can say is that my version of Psychostats works on the most recent versions of a LAMP stack + Perl.  The support for Firearms and TFC in the version that RC is hosting is very basic.  I'm trying to improve on that.  I've made quite a bit of headway but right now I'm banging my head against a wall when it comes to adding support for Firearms ffkills, which are handled differently in the logs than they are in any other mod.  Firearms has a very specific TEAMKILL event.

I've captured the event, I've got it into the database, but I'm having a hell of a time getting it from the backend to the front end, and I don't know why... :)

**Comment by Drek282** [comment]
What I've done is more than just "some fixes".  It was weeks of trial and error and a lot of minor changes throughout the front end.  However, as far as I know, all of the deprecated warnings and the errors due to obsolete code are gone.

**Comment by Drek282** [comment]
If Stormtrooper had been doing it, it probably would have taken him an hour or two, but because I'm a noob, it took weeks.

**Comment by Drek282** [comment]
Due to the fact that the existing PsychoStats repositories have either been inactive for years, or aren't doing what I'm doing, which is working with the original code and trying to add support for other games and mods, I have created a repository for what I've been working on.

You can find it here:
https://github.com/Drek282/PsychoStats
